packages/formstr-app/src/containers/PollResponse/FetchResults.tsx

```diff
@@ -0,0 +1,91 @@
+import { Filter } from "nostr-tools";
+import { Event } from "nostr-tools";
+import { pollRelays } from "../../nostr/common";
+import { useEffect, useState } from "react";
+import { Analytics } from "../PollResults/Analytics";
+import { SubCloser } from "nostr-tools";
+import { nip13 } from "nostr-tools";
+import { useApplicationContext } from "../../hooks/useApplicationContext";
+
+interface FetchResultsProps {
+  pollEvent: Event;
+  filterPubkeys?: string[];
+  difficulty?: number;
+}
+export const FetchResults: React.FC<FetchResultsProps> = ({
+  pollEvent,
+  filterPubkeys,
+  difficulty,
+}) => {
+  const [respones, setResponses] = useState<Event[] | undefined>();
+  const [closer, setCloser] = useState<SubCloser | undefined>();
+  const relays = pollEvent.tags
+    .filter((t) => t[0] === "relay")
+    ?.map((r) => r[1]);
+  const pollExpiration = pollEvent.tags.filter(
+    (t) => t[0] === "endsAt"
+  )?.[0]?.[1];
+  const { poolRef } = useApplicationContext();
+  const getUniqueLatestEvents = (events: Event[]) => {
+    const eventMap = new Map<string, Event>();
+
+    events.forEach((event) => {
+      if (
+        !eventMap.has(event.pubkey) ||
+        event.created_at > eventMap.get(event.pubkey)!.created_at
+      ) {
+        if (difficulty && nip13.getPow(event.id) < difficulty) {
+          return;
+        }
+        eventMap.set(event.pubkey, event);
+      }
+    });
+
+    return Array.from(eventMap.values());
+  };
+
+  const handleResultEvent = (event: Event) => {
+    setResponses((prevResponses) => [...(prevResponses || []), event]);
+  };
+
+  const fetchVoteEvents = (filterPubkeys: string[]) => {
+    if (closer) {
+      closer.close();
+      setResponses(undefined);
+    }
+    let resultFilter: Filter = {
+      "#e": [pollEvent.id],
+      kinds: [1070, 1018],
```

Comment:
according to the nip it's only 1018, i think 1070 is a mistake here: 

https://github.com/nostr-protocol/nips/blob/master/88.md